src/renderer/services/thorchain/thorNode.ts

```diff
@@ -143,13 +144,20 @@ const getLiquidityProviders = ({
         RxOp.map(A.filterMap(FP.flow(LiquidityProviderIO.decode, O.fromEither))),
         RxOp.map(RD.success),
         liveData.map(
-          A.map((provider) => ({
-            asset: provider.asset,
-            address: provider.rune_address,
-            assetAddress: provider.asset_address,
-            pendingRune: baseAmount(provider.pending_rune, THORCHAIN_DECIMAL),
-            pendingAsset: baseAmount(provider.pending_asset, assetDecimal)
-          }))
+          A.map((provider): LiquidityProvider => {
+            const pendingRuneAmount = baseAmount(provider.pending_rune, THORCHAIN_DECIMAL)
+            const pendingAssetAmount = baseAmount(provider.pending_asset, assetDecimal)
```

Comment:
just realized that such transformation to the `BaseAmount` is a little bit incorrect... `assetDecimal` is not always the same as `selectedPoolAsset`'s decimal in general.
for the future i believe we need to pass pending amounts as strings directly and transform to the `BaseAmount` only "on demand" (at the view level when we know assets' decimals...)

Reply:
Good point!

Reply:
_Will be done with another PR - see #1512_